spec/email_notification_spec.rb

```diff
@@ -9,21 +11,56 @@
   end
 
   describe 'POSTing a signup Notification to /user-signup/email-notification' do
-    it 'returns a 200' do
+    let(:from_address) { 'dummy@example.com' }
```

Comment:
should this succeed given it's not an authorised domain?

Reply:
Succeed in what sense, just wanted to put in a dummy value for default.  The later tests set up specific values when asserting if `User#generate` has been called or not.